src/pages/NewChatPage.js

```diff
@@ -77,10 +80,17 @@ class NewChatPage extends Component {
     getSections() {
         const sections = [];
 
+        sections.push({
+            title: this.props.translate('iou.recents'),
```

Comment:
Now that we're using this String for non-IOU pages, I think it should be refactored to a different group. Could you move the `iou.contacts` and `iou.recents` to `common.contacts` and `common.recents` please.

Reply:
Yes I moved